samples/MvcSample.Web/HomeController.cs

```diff
@@ -90,6 +90,18 @@ public ActionResult SaveUser(User user)
         }
 
         /// <summary>
+        /// Action that shows multiple file upload.
+        /// </summary>
+        public ActionResult PostFile(IEnumerable<IFormFile> file)
```

Comment:
Also can you add the following action types:
- action having a parameter which is a complex type having regular properties + files (example: UserProfile type having Name, BlogUrl, Photo properties)..here Photo is of type IFormFIle
- action having parameters (name, blogurl, photo)